### PR TITLE
updated example plugin to use version 2.0.2 of crate

### PR DESCRIPTION
**Pull request by dornbirndevelops**

### CODE REVIEW

File: src/main/java/io/crate/plugin/ExamplePlugin.java [hunk 1]
```diff
@@ -1,62 +1,56 @@
...
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
- * License for the specific language governing permissions and limitations
- * under the License.
- *
- * However, if you have executed another commercial license agreement
- * with Crate these terms will supersede the license and you may use the
- * software solely pursuant to the terms of the relevant commercial agreement.
- */
+//
```
Comment (mikethebeer) [inline src/main/java/io/crate/plugin/ExamplePlugin.java]: pls use the CRATE license header

**Review by mikethebeer — CHANGES_REQUESTED** [review]
pls squash at the end

File: src/test/java/io/crate/operation/scalar/IsEvenScalarFunctionTest.java [hunk 2]
```diff
@@ -59,71 +62,60 @@
...
+        long unevenNumber = 888888883;
+        long signedEvenNumber = -4;
+        long signedUnevenNumber = -5;
+
+        String evenText = "14";
+        String unevenText = "27";
+
+        String invalidText = "sunny";
+
+        assertTrue(isEvenFunction.evaluate((Input<Long>) () -> new Long(evenNumber)));
```
Comment (mikethebeer) [inline src/test/java/io/crate/operation/scalar/IsEvenScalarFunctionTest.java]: `assertTrue(isEvenFunction.evaluate((Input<Long>) () -> evenNumber));`

File: src/test/java/io/crate/operation/scalar/IsEvenScalarFunctionTest.java [hunk 3]
```diff
@@ -59,71 +62,60 @@
...
+        assertFalse(isEvenFunction.evaluate((Input<Long>) () -> new Long(unevenNumber)));
+        assertTrue(isEvenFunction.evaluate((Input<Long>) () -> new Long(signedEvenNumber)));
+        assertFalse(isEvenFunction.evaluate((Input<Long>) () -> new Long(signedUnevenNumber)));
+
+        assertTrue(isEvenFunction.evaluate((Input<Long>) () -> new Long(evenText)));
+        assertFalse(isEvenFunction.evaluate((Input<Long>) () -> new Long(unevenText)));
+
+        boolean exceptionThrown = false;
+        try {
```
Comment (mikethebeer) [inline src/test/java/io/crate/operation/scalar/IsEvenScalarFunctionTest.java]: remove

**Review by mikethebeer — CHANGES_REQUESTED** [review]

File: src/test/java/io/crate/operation/scalar/IsEvenScalarFunctionTest.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        long oddNumber = 888888883;
+        long signedEvenNumber = -4;
+        long signedOddNumber = -5;
+
+        assertTrue(isEvenFunction.evaluate((Input<Long>) () -> evenNumber));
+        assertFalse(isEvenFunction.evaluate((Input<Long>) () -> oddNumber));
+        assertTrue(isEvenFunction.evaluate((Input<Long>) () -> signedEvenNumber));
+        assertFalse(isEvenFunction.evaluate((Input<Long>) () -> signedOddNumber));
+
+        try {
```
Comment (mikethebeer) [inline src/test/java/io/crate/operation/scalar/IsEvenScalarFunctionTest.java]: extract into separate test method `testInvalidNumberOfArguments`

**Review by mikethebeer — CHANGES_REQUESTED** [review]

File: src/main/java/io/crate/plugin/ExamplePlugin.java [hunk 5]
```diff
@@ -1,62 +1,72 @@
...
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
- * License for the specific language governing permissions and limitations
- * under the License.
- *
- * However, if you have executed another commercial license agreement
- * with Crate these terms will supersede the license and you may use the
- * software solely pursuant to the terms of the relevant commercial agreement.
- */
+ - * Licensed to CRATE Technology GmbH ("Crate") under one or more contributor
```
Comment (mikethebeer) [inline src/main/java/io/crate/plugin/ExamplePlugin.java]: pls check `-`

**Review by mikethebeer — APPROVED** [review]
LGTM

File: build.gradle [hunk 6]
```diff
@@ -13,11 +13,12 @@ configurations {
 }
 
 dependencies {
-    compile 'io.crate:crate:1.0.2'
+    compile 'io.crate:crate:2.0.2'
+    compile 'org.apache.logging.log4j:log4j-api:2.7'
     dependency 'net.rtme:classnamerj:0.0.1'
```
Comment (seut) [inline build.gradle:18]: as you replaced the nice classnamer scalar with a boring is_even scalar ;( this dependency is no longer needed.

**Pull request merged**